packages/circuit-ui/components/Table/components/TableHead/TableHead.tsx

```diff
@@ -14,18 +14,79 @@
  */
 
 import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
-import styled from '@emotion/styled';
 import { css } from '@emotion/core';
 
 import TableRow from '../TableRow';
 import TableHeader from '../TableHeader';
 import TableCell from '../TableCell';
-import { mapCellProps, getCellChildren, RowPropType } from '../../utils';
-import { ASCENDING, DESCENDING, TH_KEY_PREFIX } from '../../constants';
+import { mapCellProps, getCellChildren } from '../../utils';
+import { Cell, Direction } from '../../types';
+import { TH_KEY_PREFIX } from '../../constants';
+import styled, { StyleProps } from '../../../../styles/styled';
 
-const fixedStyles = ({ scrollable, top, rowHeaders, theme }) =>
+type ScrollableOptions =
```

Comment:
@connor-baer I've left this as a union type, let me know if you think this should rather be a generic type! I think this implementation works well (TypeScript gives a warning when `scrollable` is set without `top` for example)

Reply:
Let's leave it like this, it works and seems simpler 👍🏻